py/bitbox02/u2fhid/u2fhid.py

```diff
@@ -84,9 +84,17 @@ def write(hid_device: SupportsReadWrite, data: bytes, cmd: int, cid: int) -> Non
         data: Data to send
         cmd: U2F HID command
         cid: U2F HID channel ID
+    Throws:
+        ValueError: In case any value is out of range
     """
+    if cmd < 0 or cmd > 0xFF:
+        raise ValueError("Channel command is out of range '0 > cmd > 0xFF'")
```

Comment:
you mean `0 <= cmd <= 0xFF`? Fyi, you can use this as a valid Python expression as well:

`if not (0 <= cmd <= 0xFF): ...`

Reply:
Yeah the exception had the wrong comment. Oh, I forgot about that. Anyway I prefer the current syntax. The python syntax is a bit weird.